src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
There is not reason to extend lottery from ColorSupplier, they are 2 totally different classes with different purpose, so they should not be in one hierarchy. So I suggest you to just create and object of ColorSupplier in Lottery and use getRandomColor method